MANIFEST.in

```diff
@@ -1,37 +0,0 @@
-# Things to always exclude
-global-exclude .git*
-global-exclude .ipynb_checkpoints
-global-exclude *.py[co]
-global-exclude __pycache__/**
-
-# Top-level Config
-include LICENSE
-include MANIFEST.in
-include setup.cfg
-include versioneer.py
-include requirements.txt
-include requirements-dev.txt
-include requirements-client.txt
-include src/prefect/_version.py
-include src/prefect/py.typed
-include src/prefect/settings/profiles.toml
-include src/prefect/deployments/recipes/*/*.yaml
-include src/prefect/deployments/templates/*.yaml
-include src/prefect/.prefectignore
-include src/prefect/logging/logging.yml
-include src/prefect/cli/templates/*.yaml
-include src/prefect/server/collection_blocks_data.json
-include src/prefect/server/api/collections_data/views/*.json
-
-# Migrations
-include src/prefect/server/database/alembic.ini
```

Comment:
I couldn't find where these sorts of files get configured in the new setup?

Reply:
We don't need explicitly list these for inclusion any longer. Anything under `src/prefect` gets scooped into the build.

I think CI is a little bit faster with these changes.